sdks/java/fn-execution/build.gradle

```diff
@@ -37,6 +37,7 @@ dependencies {
   shadow library.java.grpc_stub
   shadow library.java.grpc_netty
   shadow library.java.netty_transport_native_epoll
+  provided library.java.junit
```

Comment:
Is it a mistake or what is the sense of having it provided and testCompile too?

Reply:
testCompile only applies to the test source set.
provided applies to the main source set, there are some test utility classes in the main source set that rely on junit classes.

Reply:
Well my point was about actually the next line where the junit dependency is repeated but as `testCompile` which is unneeded given that it is now covered by `provided`.

Reply:
(actually I made a mistake in the comment it was 'provided seems enough'